### ISSUE TITLE
update the bmark get api call with tag suggestions from readable body like the web edit page

### ISSUE THREAD

**Issue by mitechie** [bookieio/Bookie#407]

**Comment by rakeshmahadasa** [comment]
I need a little help here , As we are talking about the tags at bmark_get api , it must be for chrome suggestions and as we dont have the readable at that time how will we suggest tags based on readable or do you want me to add code HERE [https://github.com/bookieio/Bookie/blob/develop/bookie/views/api.py#L193]

and yeah i will add the code to utils!!

**Comment by mitechie** [comment]
You're right that the best way to do this is some cleanup. We should first remove the 'last' code and the idea of recent tag suggestions and stick with the real suggestions generated. 

Then, this api call should come with a new query string flag 'with_suggestions' and when used, it would check if the bookmark found has a readable entry, and if so, suggest based on that content. 

So this bug has some pre-req's that need to get cleared out first to do it correctly.

**Comment by rakeshmahadasa** [comment]
i will start with removing 'last' code!!
